homeassistant/components/xiaomi_miio/const.py

```diff
@@ -330,9 +336,18 @@ class SetupException(Exception):
 FEATURE_FLAGS_AIRPURIFIER_MIOT = (
     FEATURE_SET_BUZZER
     | FEATURE_SET_CHILD_LOCK
-    | FEATURE_SET_FAVORITE_LEVEL
+    | FEATURE_SET_FAVORITE_LEVEL_MIOT
```

Comment:
Why do we change the favorite level feature  set here?

Reply:
Bump.

Reply:
In the python-miio library the set_favorite_level function for MIOT purifier checks different levels(1-14) compared to the default(1-17). Would trigger exception for levels bigger than 14.

Reply:
If this is a bug fix, please move the change to a separate PR. We want to separate fixes from new features. Each PR should only do one thing. Thanks!

Reply:
ok, but it is also needed for this PR because otherwise exceptions would be thrown for the purifier 4 on level change.

Reply:
We can merge the fix PR quickly.

Reply:
I just created the PR 75814 for that, please check. I will merge with this one once that one is on dev.

Reply:
pushed the update with the remove of FEATURE_SET_FAVORITE_LEVEL_MIOT